modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20241,7 +20254,9 @@ x-has-shared-x-with-you-for-x-until-x={0} has shared {1} with you for {2} until
 x-has-x-connections={0} has {1} connections.
 x-has-x-friends={0} has {1} friends.
 x-has-x-object-entries="{0}" has {1} object entries.
+x-has-x-object-definitions="{0}" has {1} object definitions.
 x-history={0} History
+deleting-an-objects-folder-will-move-its-object-definitions=Deleting an Objects Folder will move its Object definitions to the Uncategorized model. This action is permanent and cannot be undone.
```

Comment:
@rebsilva Shouldn't it be "... to the Uncategorized folder"? Please, check with the designer.

Reply:
@rebsilva Same comment about squashing :)